src/game.c

```diff
@@ -204,3 +205,19 @@ void handle_computer_turn(void)
     return;
 }
 
+/**
+ * @brief Draw a new card and add it to another player's cards on hand list.
+ *
+ * @param player  The player who discards a draw one card.
+ * @return int 0 - Successful;
+ *             1 - Failed due to error in malloc;
+ */
+int player_process_draw_one_card(PlayerType_e player)
+{
+    PlayerType_e be_applied_player = (player + 1) % PLAYERS_NUM;
+    Card_t draw_card = draw_one_card();
+    int ret = add_card_at_end(g_players[be_applied_player].cards_on_hand, draw_card);
+    printf("Apply Draw One Card to %s.\n", PLAYER_TYPE_STRING[be_applied_player]);
```

Comment:
Can you add this to the print message to say that a 'particular player dropped a draw one card' and then 'adding a card to the next player'

Reply:
Done.